gpflow/params/parameter.py

```diff
@@ -452,48 +460,6 @@ def _set_parameter_attribute(self, attr, value):
             raise GPflowError(msg.format(name, attr.interface))
         object.__setattr__(self, name, value)
 
```

Comment:
Could you explain what  _html_table_rows was being used for? It seems to be mentioned in .covargerc only on main.

Reply:
Guess, it was used for printing in notebooks. It looks like another dangled code, which we forgot to delete.